File: s3/buckets/templates/1/fortran/code/main.f90
program hello
	character(len=64) :: arg
	character(len=64) :: data

	narg = command_argument_count()
	if (narg > 0) then
		call get_command_argument(1,arg)
		open(unit=1, file=arg)
		read(1,'(a)') data
		write(*,'(a,a)')"Hello ",data
		close(unit=1)

		open(unit=1, file='../results/result.txt')
		write(1,'(a,a)')"Hello ",data
		close(unit=1)
	end if
end

File: s3/buckets/templates/1/perl/code/main.pl
if (open(my $ifh, '<:encoding(UTF-8)', $ARGV[0])) {
    my $data = <$ifh>;
    print "Hello $data";
    close $ifh;
    if (open(my $ofh, '>:encoding(UTF-8)', '../results/result.txt')) {
        print $ofh "Hello $data";
        close $ofh;
    }
}
